frontend/src/components/IngredientItem.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { makeStyles } from "@material-ui/core/styles";
+import { Chip } from "@material-ui/core";
+
+export default function IngredientItem({ ingredient }) {
+  const useStyles = makeStyles((theme) => ({
+    root: {
+      display: "flex",
+      justifyContent: "center",
+      flexWrap: "wrap",
+      "& > *": {
+        margin: theme.spacing(0.5),
+      },
+    },
+  }));
+
+  const classes = useStyles();
+
+  const handleDelete = () => {
+    console.log("Delete wurde gedrückt");
```

Comment:
I would not implement delete functions/ components to this "Add ingredients" PR

Reply:
This will be moved to a component DeleteIngredients